### ISSUE TITLE
explode(...) misused

### ISSUE THREAD

**Issue by michabbb** [kalessil/phpinspectionsea#839]
lets say i have a string `a,b,c` and i need to count the elements (split by ,) i would do (as usual)

```php
$array = explode(',', 'a,b,c');
$arrayCount = count($array);
```
but the inspection says "misused". because this is most simple way for me, i would like to know a version
that is "okay" for the inspection ;)

many thanks

**Comment by kalessil** [comment]
Hi Michael,

this one =): 
![image](https://user-images.githubusercontent.com/1577185/36993501-4a27e298-20ae-11e8-80bf-4d54137ab1b0.png)

Cheers, Vlad

**Comment by kalessil** [comment]
In case of: `$arrayCount = count(explode(',', 'a,b,c'));` a QF is available.

**Comment by michabbb** [comment]
@kalessil many thanks, my phpstorm didn´t show me this suggestion because the count is a few rows later ;( good2know ;) btw... what is ment with the shortcut **QF** ? ;)

**Comment by kalessil** [comment]
Quick-Fixes: Alt+Enter on reported statements normally do fix code for you.

**Comment by michabbb** [comment]
okay, thanks, i know this. because explode and count were not below each other, phpstorm didn´t show me this QF ;) `you never stop learning` ;)
